src/app/components/taskForm/cardFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+type Props = {
+    submit: any;
+    taskId: number;
+    studentId: string;
+};
+
+export default class CardFooter extends React.Component<Props, any> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleRepInput = (event: any) => {
+        this.setState({ assignmentRepo: event.target.value });
+    };
+
+    handleCommentInput = (event: any) => {
+        this.setState({ studentComment: event.target.value });
+    };
+
+    render() {
+        const { submit, taskId, studentId } = this.props;
+        const { assignmentRepo, studentComment } = this.state;
+        return (
+            <small className="text-muted">
+                <form
+                    onSubmit={e => {
+                        e.preventDefault();
```

Comment:
Please, don't use arrow functions in render. See react documentation here https://reactjs.org/docs/faq-functions.html